build/NugetConfigFile.targets

```diff
@@ -0,0 +1,58 @@
+<Project ToolsVersion="15.0">
+  <Target Name="WriteNugetConfigFile">
+
+    <PropertyGroup>
+      <NugetConfigHeader>
+        &lt;?xml version=&quot;1.0&quot; encoding=&quot;utf-8&quot;?&gt;
+        &lt;configuration&gt;
+        &lt;packageSources&gt;
+        &lt;!--To inherit the global NuGet package sources remove the &lt;clear/&gt; line below --&gt;
+        &lt;clear /&gt;
+      </NugetConfigHeader>
+
+      <NugetConfigFeedsPrivate>
+        &lt;add key=&quot;PrivateBlobFeed&quot; value=&quot;$(ExternalRestoreSources)&quot; /&gt;
```

Comment:
Can there be more than one external restore source?  If so, then this probably won't work.  If not, then the variable seems named incorrectly.